### DISCUSSION TITLE
Frsky Pilot FC iNav support

### DISCUSSION THREAD

**Discussion by npirkes** [iNavFlight/inav#7926] [Q&A] [answered]
Hi,

How can I figure out which version of iNav I can use with my Frsky Pilot FC?
As I recall the product spec, its board is based on Matek F765 and it comes pre-installed with iNav 2.60.
Does it means it will support also the latest versions?

Thanks in advance!

**Comment by MrD-RC** [comment, marked as answer]
It’s in iNav _from_ version 2.6.
